### ISSUE TITLE
UI incorrectly states successful creation

### ISSUE THREAD

**Issue by lastcoolnameleft** [Microsoft/todo-app-java-on-azure#41]
I have followed the instructions provided in the repo and everything appeared to be working correctly.  I ran the `mvn package`, updated the credentials, ran `java -jar target/todo-app-java-on-azure-1.0-SNAPSHOT.jar` and then went to `localhost:8080` and created a new item (which the UI said it completed).  

Using the Data Explorer, I couldn't find any of the data.  I then changed the credentials so that I knew that they'd be wrong and found that the app continued to say it would create/update items.

In short, the UI appears to be providing incorrect information by saying that it created the item successfully.

P.S.  I even deleted the CosmosDB instance and it still claimed to successfully create the TODO item.

**Comment by yaweiw** [comment]
@yuwzho please take a look when you can.
